src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] parts = readFromFile(fromFileName).split(" ");
+        int sumBuy = getSumBuy(parts);
+        int sumSupply = getSumSupply(parts);
+        int difference = getDifference(sumBuy, sumSupply);
+        getWrite(toFileName, sumSupply, sumBuy, difference, parts);
```

Comment:
Try to make it this way
```suggestion
        List<String> data = readFile(fromFileName);
        String report = makeReport(data);
        writeToFile(report, toFileName);
```